### ISSUE TITLE
audio content is broken for learners

### ISSUE THREAD

**Issue by indirectlylit** [learningequality/kolibri#1127]
Currently, trying to play audio content results in an error

```
Uncaught TypeError: Cannot read property 'userAgent' of undefined
```

Seems to be inside of the call to this library:

```javascript
require('html5media/dist/api/1.1.8/html5media');
```

![image](https://cloud.githubusercontent.com/assets/2367265/24274163/cb9b5fe4-0fe3-11e7-804f-c3d254194afd.png)

**Comment by christianmemije** [comment]
Good time to consider https://trello.com/c/hqFja95k/579-merge-audio-and-video-js-for-audio-files-would-make-audio-player-compatible-across-all-browsers

**Comment by indirectlylit** [comment]
closing as a duplicate: #1057

**Comment by indirectlylit** [comment]
> Good time to consider https://trello.com/c/hqFja95k/579-merge-audio-and-video-js-for-audio-files-would-make-audio-player-compatible-across-all-browsers

Agree that's a good direction to go, but (just guessing here) it's probably much faster to fix this critical bug so I don't think it necessarily warrants throwing out the entire implementation yet.

**Comment by christianmemije** [comment]
Reopening as it still exists.

**Comment by rtibbles** [comment]
Fixed in #1842
